### ISSUE TITLE
Add project to ignite repository list

### ISSUE THREAD

**Issue by vfdev-5** [dmitry-vorobiev/generative-models#5]
Hi @dmitry-vorobiev 

I would like to add your project to https://github.com/pytorch/ignite#others
and optionally even create a working Colab with wandb online logs and generated images by trained model.
How would you estimate the readiness of this repository ?

I took a look on your ["Projects" kanban](https://github.com/dmitry-vorobiev/generative-models/projects/1) and I'd also be interested in adding AMP training.

What do you think about all this ?

PS: optionally you can also ping me on ODS slack (@vfdev-5) if you would like to discuss more.

**Comment by dmitry-vorobiev** [comment]
Yeah, I think it would be great :) 

I would say, the repo is ~70% ready. It's somewhat useful (I hope), but there are a few things that are still missing from the original StyleGAN2 work. I also saw several papers about improving various aspects of StyleGAN, that I would like to try and test.

The original work contains optimized AMP settings (like the loss scaling coefficients), but I didn't study them closely:
https://github.com/NVlabs/stylegan2/blob/master/dnnlib/tflib/optimizer.py
